include/engine/polyline_compressor.hpp

```diff
@@ -30,27 +30,24 @@ std::string encodePolyline(CoordVectorForwardIter begin, CoordVectorForwardIter
         return {};
     }
 
-    std::vector<int> delta_numbers;
```

Comment:
Optimisation 1: instead of having temporary `delta_numbers` array and then using `details::encode` on it we encode coordinates "on the fly".